pydantic/errors.py

```diff
@@ -49,7 +49,12 @@ class NoneIsAllowedError(PydanticTypeError):
 
 class WrongConstantError(PydanticValueError):
     code = 'const'
-    msg_template = 'expected constant value {const!r}'
+
+    def __str__(self) -> str:
+        assert self.ctx is not None
+        given = self.ctx['given']
```

Comment:
we don't include the given value in other error messages, better not to include it here for consistency.

Good idea to keep it in the context so others can use it if they wish.